lib/client.js

```diff
@@ -99,23 +100,34 @@ extend(Raven.prototype, {
       globalContext.extra = options.extra;
     }
 
+    this.onFatalError = this.defaultOnFatalError = function (err, sendErr, eventId) {
+      console.error(err.stack);
+      process.exit(1);
+    };
+
     this.on('error', function (err) {
       utils.consoleAlert('failed to send exception to sentry: ' + err.message);
     });
 
     return this;
   },
 
-  install: function install(opts, cb) {
+  install: function install(cb) {
     if (this.installed) return this;
```

Comment:
It was initially used for passing autobreadcrumb and captureUnhandledRejections configs; we decided to move that to part of `config()`, so it was unused and I just hadn't removed it.